src/main/java/core/basesyntax/MainClass.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class MainClass {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[5];
```

Comment:
5 is a magic number, let's create a constant for it

Reply:
done